splot/tests/test_viz_esda_mpl.py

```diff
@@ -197,31 +210,33 @@ def test_moran_loc_scatterplot():
                  scatter_kwds=dict(c='#4393c3'))
 
 
-def test_lisa_cluster():
-    columbus = examples.load_example('Columbus')
-    link_to_data = columbus.get_path('columbus.shp')
-    df = gpd.read_file(link_to_data)
-
-    y = df['HOVAL'].values
-    w = Queen.from_dataframe(df)
+def _test_calc_moran_loc(gdf, var='HOVAL'):
+    y = gdf[var].values
+    w = Queen.from_dataframe(gdf)
     w.transform = 'r'
-
+    
     moran_loc = Moran_Local(y, w)
+    return moran_loc
+
+
+def test_lisa_cluster():
+    df = _test_data_columbus()
+    moran_loc = _test_calc_moran_loc(df)
 
     fig, _ = lisa_cluster(moran_loc, df)
     plt.close(fig)
+    
+    # test LineStrings
+    df_line = _test_LineString()
+    moran_loc = _test_calc_moran_loc(df_line)
+    
+    fig, _ = lisa_cluster(moran_loc, df)
+    plt.close(fig)
```

Comment:
Do you want to check the actual figure?

Reply:
I am definitely open to suggestions how the figure could be checked! :)

Reply:
One option is to get the attributes from the plotted collection as we do in geopandas https://github.com/geopandas/geopandas/blob/64598b6f0d872baf09657c27a9aa4c87d86d9131/geopandas/tests/test_plotting.py#L169

The other is to keep a reference image in tests and check against that as matplotlib does in some cases https://github.com/matplotlib/matplotlib/blob/ec126b0175c657f0adb5ca635eeabad7c142bb0b/lib/matplotlib/tests/test_colorbar.py#L188-L190